extensions/MyDashboard/web/js/query.js

```diff
@@ -154,9 +169,21 @@ $(function() {
 
         bugQueryTable.plug(Y.Plugin.DataTableSort);
 
+        var auto_updateQueryTable = function(o){
+            button_state = auto_refresh.checked;
+            if(button_state == true){
+                refresh_interval = setInterval(function(e) {
+                    updateQueryTable(default_query);
+                },1000*60*10);
+            }else if(button_state == false){
+                clearInterval(refresh_interval);
+            }
```

Comment:
No need to explicitly check for false. Anything not true is false. Also can be shortened a bit.

```
            if (auto_refresh.checked == true) {
                refresh_interval = setInterval(function(e) {
                    updateQueryTable(default_query);
                },1000*60*10);
            }
            else {
                clearInterval(refresh_interval);
            }
```